server/routes/redirects.py

```diff
@@ -101,4 +106,17 @@ def insights():
   return redirect(
       url_for('explore.page',
               _scheme=current_app.config.get('SCHEME', 'https'),
-              code=302))
\ No newline at end of file
+              code=302))
+
+
+def load_redirects(name):
+  local_file = gcs.download_file(bucket=GLOBAL_CONFIG_BUCKET,
+                                 filename='redirect.json')
```

Comment:
Should we also check-in the file so we have a record of the URLs we have and can restore to GCS, etc.

Reply:
Done